projects/fsastorefrontstyles/scss/components/checkout/_mini-cart.scss

```diff
@@ -31,3 +31,37 @@
     }
   }
 }
+
+.action-wrapper {
+  display: flex;
+  justify-content: space-between;
+  cx-fs-cart-coupon {
+    width: 60%;
```

Comment:
We should use flex values since it's inside flex wrapper.

Reply:
It seems there is no need for width or flex.